indian-languages/README.md

```diff
@@ -0,0 +1,9 @@
+# Bharati Braille
+
+Most Indian languages use Bharati Braille a system developed in the 1950's to unify as far as possible the Braille codes for the various Indian languages and scripts.
+
+The documents here were supplied by Braille Council of India.
+
+`Braille chart Pg-1.jpg` and `Braille chart pg-2.jg` show a summary of the Bharati Braille system. For how it applies to individual languages refer to [World Braille Usage (3rd edition)](../world-braille-usage-third-edition.pdf).
```

Comment:
It would be good if we would know from which document these pages come (title, author, year, etc.).

Reply:
I will ask Dipendra if he can tell me. Searching on internet they look very similar to pages from World Braille Usage 1953 but with some differences.

Reply:
Could they be from a different edition of WBU?